### PR TITLE
DEV-3180-new-go-exercise-fifthandskip

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/FifthAndSkip/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# FifthAndSkip
+
+### Instructions
+
+Write a function `FifthAndSkip()` that takes a string and returns another one with words of 5 characters and skips the next character followed by newline `\n`.
```
Comment (Hamzaelkhatri) [inline subjects/FifthAndSkip/README.md:5]: The description is not clear.
> Write a function with a string in the parameter and return a new string. The function separates every five characters in the string with space and removes the six ones.

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: subjects/fifthandskip/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+## fifthandskip
+
+### Instructions
+
+Write a function `FifthAndSkip()` that takes a `string` and returns another `string`. The function separates every five characters of the `string` with a space and removes the sixth one.
+
+- If there is a space in the middle of a word it should ignore it and get the first character until getting to a length of 5.
+- If the `string` is less than 5 characters returns "Invalid Output\n".
```
Comment (jotapero) [inline subjects/fifthandskip/README.md:8]: suggestion: (...) returns "Invalid Output" followed by a newline `\n`.

Comment (jotapero) [inline subjects/fifthandskip/README.md:8]: fix: Add condition for empty string, If the `string` is empty or less than 5 ...

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/fifthandskip/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/fifthandskip/README.md:8]: It is fixed.

File: subjects/fifthandskip/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+## fifthandskip
+
+### Instructions
+
+Write a function `FifthAndSkip()` that takes a `string` and returns another `string`. The function separates every five characters of the `string` with a space and removes the sixth one.
+
+- If there is a space in the middle of a word it should ignore it and get the first character until getting to a length of 5.
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/fifthandskip/README.md:7]: Thank you, it is fixed.

File: subjects/fifthandskip/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/fifthandskip/README.md:8]: Thank you, it is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**